NetKVM/Common/ParaNdis_Common.cpp

```diff
@@ -1098,7 +1118,23 @@ static NDIS_STATUS ParaNdis_VirtIONetInit(PARANDIS_ADAPTER *pContext)
     DEBUG_ENTRY(0);
     UINT i;
     USHORT nVirtIOQueues = pContext->nHardwareQueues * 2 + 2;
-
+#if NDIS_SUPPORT_NDIS650
+    #if ALICLOUD_MAX_FRAGMENTS_IN_ONE_NB
+        for (ULONG j = 0; j < PARANDIS_SEND_COPY_PAGES; j++)
+        {
+            CNdisSharedMemory* pPage = (CNdisSharedMemory*)ParaNdis_AllocateMemory(pContext, sizeof(*pPage));
+            pPage->Create(pContext->MiniportHandle);
```

Comment:
Don't you want to fail the init in the case of failure here?

Reply:
if memory allocation fails, I don't want to fail the init.  The benefit of this is that even if it fails, it won't hinder the subsequent processes. At most, it just won't use the copy mode.

How do you perceive it,Is my approach reasonable?